composer.json

```diff
@@ -29,7 +30,7 @@
         "phpunit/php-code-coverage": "@stable",
         "symfony/form": "~2.3 || ~3.0",
         "doctrine/phpcr-odm": "~1.3,>1.3.0-rc3",
-        "symfony-cmf/testing": "^1.3.0",
+        "symfony-cmf/testing": "dev-remove_csrf_protection as 1.3.0",
```

Comment:
needs to be reverted once https://github.com/symfony-cmf/Testing/pull/119 is merged